_posts/2017-09-05-planning-for-7.0.md

```diff
@@ -0,0 +1,676 @@
+
+This means that we intend to make major version bumps to any experimental proposal plugins when the spec changes rather than waiting to update all of Babel. So anything that is < Stage 4 would be open to breaking changes in the form of major version bumps and same with the Stage presets themselves if we don't drop them entirely.
+
+For example:
+
+Say you are using preset-env (which keeps up to date and currently includes everything in es2015, es2016, es2017) + an experimental plugin. You also decide to use object-rest-spread because it's cool.
+
+```json
+{
+  "presets": ["env"],
+  "plugins": ["transform-object-rest-spread"]
+}
+```
+
+If the spec to an experimental proposal changes, we should be free to make a breaking change and make a major version bump for that plugin only. Because it only affects that plugin, it doesn't affect anything else and people are free to update when possible. We just want to make sure that users update to the latest version of any experimental proposal when possible and provide the tools to do so automatically if that is reasonable as well.
+
+### TODOs?
+
+> I believe the way we want to go about doing this is to move those packages into the `experimental/` folder in our [monorepo](https://github.com/babel/babel) instead of in `packages/`.
+> Then we should rename all proposals to `babel-plugin-proposal-` instead of `babel-plugin-transform-`
+> Change our publish process (probably through Lerna) to publish the packages in `experimental/` independently.
+
+## 💀 Possible Deprecations
+
+### Deprecate the "env" option in `.babelrc`
+
+> [babel/babel#5276](https://github.com/babel/babel/issues/5276)
+
+The "env" configuration option (not to be confused with babel-preset-env) has been a source of confusion for our users as seen by the numerous issues reported.
+
+The [current behavior](http://babeljs.io/docs/usage/babelrc/#env-option) is to merge the config values into the top level values which isn't always intuitive such that developers end up putting nothing in the top level and just duplicating all the presets/plugins under separate envs.
+
+To eliminate the confusion (and help our power users), we're considering dropping the env config option all together and recommending users use the proposed JS config format (see below).
+
+### Deprecate ES20xx presets
+
+> We already deprecated preset-latest a while ago, and ES2016/ES2017 [earlier](https://twitter.com/left_pad/status/897483806499885057)
+> It's annoying making a yearly preset (extra package/dependency, issues with npm package squatting unless we do scoped packages)
+
+Developers shouldn't even need to make the decision of what yearly preset to use? If we drop/deprecate these presets then everyone can use [babel-preset-env](https://github.com/babel/babel-preset-env) instead which will already update as the spec changes.
+
+## 🤔 Questions
+
+### ~~Deprecate/Rename/Remove Stage X presets~~
```

Comment:
I use env + stage-3 + specific plugins. stage-3 is relatively stable although I'm positive I'm only using a small percentage of the transforms there. 

Now that we've gotten through that initial "wave" of features, this may make sense. It's probably controversial though.